docs/content/en/getting_started/upgrading.md

```diff
@@ -21,6 +21,13 @@ DockerHub to update.
 
 
 The generic upgrade method for docker-compose are as follows:
+-   Pull the latest DefectDojo changes
+
+    ``` {.sourceCode .bash}
+    git fetch
+    git pull
+    git merge origin/master
+    ```    
```

Comment:
This should go down to the "Building your local images" section instead

Reply:
@Maffooch thank you for the explanation, it is much appreciated!  I have pushed a new commit with the suggested revision under "building your own images" along with a few wording adjustments to replace the mentioned glossed over step.